flare_portal/experiments/templates/experiments/researcher_delete_form.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+
+{% block title %}Delete Researcher - {{ project.name }}{% endblock title %}
```

Comment:
suggestion: add `{{ researcher.username }}` in the title?

```suggestion
{% block title %}Delete Researcher {{ researcher.username }} - {{ project.name }}{% endblock title %}
```

Reply:
Thanks!